src/coreclr/inc/icorjitinfoimpl_generated.h

```diff
@@ -417,7 +417,13 @@ CORINFO_ARG_LIST_HANDLE getArgNext(
 CorInfoTypeWithMod getArgType(
           CORINFO_SIG_INFO* sig,
           CORINFO_ARG_LIST_HANDLE args,
-          CORINFO_CLASS_HANDLE* vcTypeRet) override;
+          CORINFO_CLASS_HANDLE* vcTypeRet
+#if defined(TARGET_LOONGARCH64)
+          ,int *flags = NULL
+#endif
+) override;
+
+    uint32_t getFieldTypeByHnd(CORINFO_CLASS_HANDLE cls) override;
```

Comment:
I think in order to fix compiler errors should be added line below this. https://github.com/dotnet/runtime/blob/823ec6914c2ea5f9e99bc261c6247e4d78fa1cfd/src/coreclr/tools/Common/JitInterface/ThunkGenerator/ThunkInput.txt#L256
That's my guess based on https://github.com/dotnet/runtime/blob/main/docs/project/updating-jitinterface.md

This whole file is generated, so manual changes like this and above would not work probably.

Reply:
@kant2002
Thanks for your review～